src/client/src/ui/spotTile/model/spotTileModel.js

```diff
@@ -99,6 +104,12 @@ export default class SpotTileModel extends ModelBase {
     this._regionManagerHelper.popout(`${this.currencyPair.symbol} Spot`, 332, 155);
   }
 
+  @observeEvent('displayCurrencyChart')
+  _onDisplayCurrencyChart(){
+    this._log.info(`Display currency chart - ChartIQ`);
```

Comment:
does this line pass the linter? Thought it blows up when you use a template string without any template variables